src/cli/simple/command/mod.rs

```diff
@@ -897,6 +918,43 @@ fn str_to_identifier(id: &str) -> anyhow::Result<Erc20Identifier> {
 
 struct HexString(String);
 
+#[derive(Debug, Clone, Default, ValueEnum)]
+pub enum WaitUntil {
+    None,
+    Included,
+    ExecutedOptimistic,
+    IncludedFinal,
+    Executed,
+    #[default]
+    Final,
+}
+
+impl From<WaitUntil> for TxExecutionStatus {
+    fn from(value: WaitUntil) -> Self {
+        match value {
+            WaitUntil::None => Self::None,
+            WaitUntil::Included => Self::Included,
+            WaitUntil::ExecutedOptimistic => Self::ExecutedOptimistic,
+            WaitUntil::IncludedFinal => Self::IncludedFinal,
+            WaitUntil::Executed => Self::Executed,
+            WaitUntil::Final => Self::Final,
+        }
+    }
+}
+
+impl Display for WaitUntil {
```

Comment:
Is the presence of the Display implementation required by clap?

Reply:
Yes, `default_value_t` requires to implement `ToString` trait, but I used `Display` trait because clippy recommends it instead of `ToString`